<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="currentUserLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current User:</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentUserPuzzleRatingLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Puzzle Rating: </string>
   </property>
  </widget>
  <widget class="QLabel" name="dailyPuzzleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Daily Puzzle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="matchupLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Niemann Vs Carlsen</string>
   </property>
  </widget>
  <widget class="QLabel" name="puzzleRatingLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rating:</string>
   </property>
  </widget>
  <widget class="QLabel" name="outcomeDeltaLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Outcome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="deltaLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Success +100 / Loss -200</string>
   </property>
  </widget>
  <widget class="QLabel" name="tournamentLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tournament</string>
   </property>
  </widget>
  <widget class="QLabel" name="movesLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Moves to Make:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
